code/framework/src/scripting/resource/resource_manager.cpp

```
@@ -0,0 +1,1483 @@
+                    }
+                }
+            }
+        } else {
+            Logging::GetLogger(FRAMEWORK_INNER_SCRIPTING)->warn("Resources directory not found: {}", _config.resourcesPath);
+        }
+
+        // Build dependency graph
+        BuildDependencyGraph();
+
+        Logging::GetLogger(FRAMEWORK_INNER_SCRIPTING)->info("Discovered {} resources", _resources.size());
+        return _resources.size();
```

Cppcheck (2.18.0)

[error] failed to evaluate #if condition

(preprocessorErrorDirective)